cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-westend/src/tests/asset_transfers.rs

```diff
@@ -0,0 +1,155 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::*;
+
+fn create_wnd_foreign_asset_on_asset_hub_rococo() -> MultiLocation {
+	let sudo_origin = <AssetHubRococo as Chain>::RuntimeOrigin::root();
+	let alice: AccountId = AssetHubRococo::account_id_of(ALICE);
+	let wnd_at_asset_hub_rococo =
+		MultiLocation { parents: 2, interior: X1(GlobalConsensus(NetworkId::Westend)) };
+
+	AssetHubRococo::execute_with(|| {
+		assert_ok!(<AssetHubRococo as AssetHubRococoPallet>::ForeignAssets::force_create(
+			sudo_origin,
+			wnd_at_asset_hub_rococo,
+			alice.clone().into(),
+			true,
+			ASSET_MIN_BALANCE,
+		));
+		assert!(<AssetHubRococo as AssetHubRococoPallet>::ForeignAssets::asset_exists(
+			wnd_at_asset_hub_rococo
+		));
+		type RuntimeEvent = <AssetHubRococo as Chain>::RuntimeEvent;
+		assert_expected_events!(
+			AssetHubRococo,
+			vec![
+				RuntimeEvent::ForeignAssets(pallet_assets::Event::ForceCreated {
+					asset_id,
+					..
+				}) => { asset_id: *asset_id == wnd_at_asset_hub_rococo, },
+			]
+		);
+	});
+	wnd_at_asset_hub_rococo
+}
+
+#[test]
+fn send_wnds_from_asset_hub_westend_to_asset_hub_rococo() {
```

Comment:
@acatangiu @NachoPal 
actually, this test looked better under `rococo-westend` directory, like I tried to say [here](https://github.com/paritytech/polkadot-sdk/pull/2251#issuecomment-1808097043) and Adrian was right [here](https://github.com/paritytech/polkadot-sdk/pull/2251#issuecomment-1803948590) with `tests/bridges/rococo-westend`
BH does not need to know about all use cases and combination.

Reply:
too late, I asked and didn't get clear preference other than Nacho's so went with that :) this is fine too - functionally it's the same thing

Reply:
I'll maybe refactor again at the end on reviews on PR if there is consensus to do so - otherwise will stay as is